### ISSUE TITLE
Implement a push notifications system (PNS)

### ISSUE THREAD

**Issue by stevevance** [ChicagoCityscape/website#78]
Using the existing notifications subscriptions that people are already using (Places, Companies, Address Snapshots), notifications could appear on the website in a news feed instead of or in addition to an email. For emails, records are only checked once a day, at 8 AM, but with a website-wide PNS the records could be checked more often. 

A "master table" of all of the Places, Companies, and Address Snapshots that people subscribe to would need to be created. This master table would then be updated with pertinent information like:
- timestamp of last permit issued there
- timestamp of last business license issued there
- timestamp of last filming permit issued there

Every 15 minutes, or so, the these master tables would be updated with those timestamps. Then, once that update is done, the PNS would check those master tables to see if any of the values have timestamps that occurred since the last time the PNS checked.

**Comment by stevevance** [comment]
Possible libraries to use:
- [Stream.io](https://getstream.io)

**Comment by stevevance** [comment]
This is not a priority so I'm closing it.
